packages/react-router-dom/docs/api/Link.md

```diff
@@ -10,7 +10,7 @@ import { Link } from 'react-router-dom'
 
 ## to: string
 
-A string representation of the location to link to, created by concatenating the location's pathname, search, and hash properties.
+A string representation of the link location, created by concatenating the location's pathname, search, and hash properties.
```

Comment:
You've changed "link" from a verb to a noun. In this case, a proper noun because the Link is a component type. So, it should be capitalized:

```suggestion
A string representation of the Link location, created by concatenating the location's pathname, search, and hash properties.
```